README.md

```diff
@@ -433,6 +433,9 @@ Please make sure you understand the implications of this.
 
 ## Licenses
 
+The main bulk of Krun is licensed under the UPL license, which is found at the
```

Comment:
This section is verbose and confusing. I'd suggest either removing it or saying "Krun is licensed under the UPL. However, the nbody benchmark is published under a ..." or similar.